scrapers/scrape_common.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+# Various small utilities used by scrapers.
+
+import datetime
+import os
+import subprocess
+import re
+
+def download(url, encoding='utf-8'):
+  """curl like"""
+  downloader = os.path.join(os.path.dirname(__file__), 'download.sh')
+  return subprocess.run([downloader, url], capture_output=True).stdout.decode(encoding)
```

Comment:
Rather than spawning a subprocess, I'd suggest to use the requests library, it works cross-platform, provides nice error handling (like getting the HTTP Status code if something fails) etc. the download.sh was nice for the shell scripts, but is imho not the right tool for a python-based scraper.

Reply:
This was done intentionally to share the downloader.

Reply:
Fair point, we can refactor this later. 

From an error handling point-of-view it's so much more convenient to something like requests to download the content.